Makefile

```diff
@@ -8,6 +8,7 @@ mypy:
 	mypy --ignore-missing-imports $(MODULES)
 
 test: lint mypy
+	tests/fixtures/populate.py --s3-bucket $(DSS_S3_BUCKET_TEST_FIXTURES) --gs-bucket $(DSS_GS_BUCKET_TEST_FIXTURES)
```

Comment:
The branch used for #245 (mbaumann-indexer-review-fixes) had a lot of indexer changes on it needed for the notification work. Due to the time it was taking to get the #245 branch on master, the notification work (mbaumann-notification) was recently branched from mbaumann-indexer-review-fixes (hence the change above). I think the best way to get things into a good state is to finishing landing mbaumann-indexer-review-fixes on master, merge master into mbaumann-notification, then land this branch on master. This is a top priority.